descriptors/ModuleDescriptor-template.json

```diff
@@ -463,11 +463,11 @@
     },
     {
       "id": "fqm-query",
-      "version": "1.0"
+      "version": "1.0 2.0"
```

Comment:
can it work with 1.0 as well or only with 2.0?

Reply:
I believe it will work with 1.0, but I wanted to run it by the team to be sure. The only breaking changes to the API itself are in the structure of the entity types returned by `/entity-types`. Those changes are for the entity type configuration that FQM uses internally (that part of the FQM API is leaky. We've got a ticket to hide those internal implementation details, but haven't done it yet), so they shouldn't impact consumers. Specifically, the `fromClause` property in the JSON has been replaced by `sources`, along with some changes in the structure of the `columns` array (a new `sourceAlias` property and the syntax for `valueGetter`/`filterValueGetter` changed).

The more significant breaking change isn't to the API itself, but instead breaks the data used by and exposed by the API: we changed almost every entity type ID and name, and renamed almost every field within the entity types. As a result, pretty much every stored query will need to be updated, but I don't think mod-bulk-operations has any stored queries. I think bulk edit _does_ do some custom translations for entity type names, though, which would have to based on entity type IDs or names. That will need updated if it hasn't been done yet, but I'm guessing that's in the UI repo, not the BE module.

If I'm correct about mod-bulk-operations not using any of those JSON fields in the entity type and not having any hard-coded or stored queries, then it should still be compatible with 1.0. If it _does_ have any hard-coded queries, those will need to be updated and I'd say it's not compatible. But if it has any stored queries, it should still be backward-compatible (a data migration will be needed, though. We're working on a solution in mod-fqm-manager to update those automatically).